docs/demo.mdx

```diff
@@ -0,0 +1,372 @@
+---
+title: Internal Demo Page for Styling
+---
+
+import '@site/node_modules/infima/dist/css/default/default.css';
+
+import FlavorizedCodeBlock from '@site/src/components/FlavorizedCodeBlock';
```

Comment:
```suggestion
```

Reply:
@noomorph - I removed it and replaced the FlavorizedCodeBlock wrapper that was being used inside of the INFO admonission and replaced it with a ```diff

Thanks